src/lake/Lake/Util/StoreInsts.lean

```diff
@@ -13,19 +13,19 @@ namespace Lake
 
 instance [Monad m] [EqOfCmpWrt κ β cmp] : MonadDStore κ β (StateT (DRBMap κ β cmp) m) where
   fetch? k := return (← get).find? k
-  store k a :=  modify (·.insert k a)
+  store k a := modify (·.insert k a)
 
 instance [Monad m] : MonadStore κ α (StateT (RBMap κ α cmp) m) where
   fetch? k := return (← get).find? k
   store k a := modify (·.insert k a)
 
 instance [Monad m] : MonadStore κ α (StateT (RBArray κ α cmp) m) where
   fetch? k := return (← get).find? k
-  store k a :=  modify (·.insert k a)
+  store k a := modify (·.insert k a)
 
 instance [Monad m] : MonadStore Name α (StateT (NameMap α) m) :=
   inferInstanceAs (MonadStore _ _ (StateT (RBMap ..) _))
 
-@[inline] instance [MonadDStore κ β m] [t : FamilyOut β k α] : MonadStore1 k α m where
-  fetch? := cast (by rw [t.family_key_eq_type]) <| fetch? (m := m) k
+@[inline] instance [MonadDStore κ β m] [t : FamilyOut β k α] : MonadStore1Of k α m where
+  fetch? := t.family_key_eq_type ▸ fetch? k
```

Comment:
I just noticed this, but why did you restyle this? I think consistently using `cast` over other proof tools  when needed makes more sense.

Reply:
ok, I'll revert